.github/workflows/ci-app-tests.yml

```diff
@@ -102,8 +102,7 @@ jobs:
       - name: Adjust examples
         if: ${{ matrix.pkg-name != 'lightning' }}
         run: |
-          python .actions/assistant.py copy_replace_imports --source_dir="./examples" --source_import="lightning.app" --target_import="lightning_app"
-          python .actions/assistant.py copy_replace_imports --source_dir="./examples" --source_import="lightning" --target_import="lightning_app"
+          python .actions/assistant.py copy_replace_imports --source_dir="./examples" --source_import="lightning.app,lightning" --target_import="lightning_app,lightning_app"
```

Comment:
This was not needed, U remember @carmocca telling me not to edit what is unrelated the other day :D

Reply:
Fair. I'll revert it. I believe I had other changes in this file originally, but were removed

Reply:
I'll leave it since it avoids calling the function twice unnecesarily